DOCS.md

```diff
@@ -1,11 +1,42 @@
 Use this plugin for deplying an application to Cloud Foundry. You can override
 the default configuration with the following parameters:
 
-* `` -
-
 ## Example
 
 ```yaml
 deploy:
   cloudfoundry:
+    api: api.run.pivotal.io
+    user: $$USER
+    password: $$PASSWORD
+    org: $$USER
+    space: production
+    when:
+      branch: master
 ```
+
+## Available options
```

Comment:
I would prefer the same documentation as for the other plugins with the removed list above. Beside that please use only underscores and no dashes for the config variables.